### PR TITLE
[CAFV-211] Update upgrade process in CAPVCD 1.1.0 documentation

### PR DESCRIPTION
**Pull request by ymo24**
## Description
Please provide a brief description of the changes proposed in this Pull Request

-  Update upgrade process in CAPVCD 1.1.0 documentation

## Checklist
- [x] tested locally
- [ ] updated any relevant dependencies
- [ ] updated any relevant documentation or examples

## API Changes
Are there API changes?
- [ ] Yes
- [x] No

If yes, please fill in the below

1. Updated conversions?
    - [ ] Yes
    - [ ] No
    - [ ] N/A
2. Updated CRDs?
    - [ ] Yes
    - [ ] No
    - [ ] N/A
3. Updated infrastructure-components.yaml?
    - [ ] Yes
    - [ ] No
    - [ ] N/A
4. Updated `./examples/capi-quickstart.yaml`?
    - [ ] Yes
    - [ ] No
    - [ ] N/A
5. Updated necessary files under `./infrastructure-vcd/v1.0.0/`?
   - [ ] Yes
   - [ ] No
   - [ ] N/A

## Issue
If applicable, please reference the relevant issue

Fixes #

- - -
This change is [Reviewable](https://reviewable.io/reviews/vmware/cluster-api-provider-cloud-director/426)

### CODE REVIEW

**Review by Anirudh9794 — APPROVED** [review]
> Reviewed 3 of 3 files at r1, all commit messages.
> *[Reviewable](https://reviewable.io/reviews/vmware/cluster-api-provider-cloud-director/426)* status: :shipit: complete! all files reviewed, all discussions resolved (waiting on @ymo24)

File: docs/UPGRADES.md [hunk 1]
```diff
@@ -1,8 +1,8 @@
 # CAPVCD Upgrades
 
-## Upgrades from CAPVCD 0.5.0 to 1.0.0
```
Comment (rocknes) [inline docs/UPGRADES.md:3]: We shouldn't overwrite older upgrade documentation, rather add new upgrade paths to the doc

**Review by ymo24 — COMMENTED** [review]
> *[Reviewable](https://reviewable.io/reviews/vmware/cluster-api-provider-cloud-director/426)* status: 2 of 3 files reviewed, 1 unresolved discussion (waiting on @Anirudh9794 and @rocknes)

___
*[`docs/UPGRADES.md` line 0 at r1](https://reviewable.io/reviews/vmware/cluster-api-provider-cloud-director/426#gh-1160853612:-NSRxam9BGGtUhFmElt3:bqieia3) ([raw file](https://github.com/vmware/cluster-api-provider-cloud-director/blob/e141aeb3a63ec1fb46405822a3a69ffe669e730a/docs/UPGRADES.md#L0)):*
<details><summary><i>Previously, rocknes wrote…</i></summary><blockquote>

*(Reviewable was [unable](https://github.com/Reviewable/Reviewable/issues/14) to map this GitHub inline comment thread to the right spot — sorry!)*

We shouldn't overwrite older upgrade documentation, rather add new upgrade paths to the doc
</blockquote></details>

Already add new upgrade paths (0.5.x to 1.1.0 AND 1.0.0 to 1.1.0). Thanks Aritra!

**Review by sahithi — CHANGES_REQUESTED** [review]
> *[Reviewable](https://reviewable.io/reviews/vmware/cluster-api-provider-cloud-director/426)* status: 2 of 3 files reviewed, 2 unresolved discussions (waiting on @Anirudh9794, @rocknes, and @ymo24)

___
*[`docs/UPGRADES.md` line 13 at r2](https://reviewable.io/reviews/vmware/cluster-api-provider-cloud-director/426#-NSl5DpBEMg6S5UZulR8:-NSl5DpBEMg6S5UZulR9:bxb73a2) ([raw file](https://github.com/vmware/cluster-api-provider-cloud-director/blob/622006fa4cd181dd8a53bd46a5fafdab67c34198/docs/UPGRADES.md#L13)):*
> ```Markdown
> 
> 1. Delete the capvcd 0.5.x deployment - `kubectl delete deployment -n capvcd-system capvcd-controller-manager`
> 2. Run `clusterctl upgrade apply --contract v1beta1`
> ```

What does v1beta1 represent here?
Will capvcd crds of v1beta2 they be registered or not?

**Review by sahithi — APPROVED** [review]
> Reviewed 2 of 3 files at r1, 1 of 1 files at r2, all commit messages.
> *[Reviewable](https://reviewable.io/reviews/vmware/cluster-api-provider-cloud-director/426)* status: all files reviewed, 2 unresolved discussions (waiting on @rocknes)

**Pull request merged**